plugin/src/test/kotlin/com/ritense/plugin/service/PluginServiceIT.kt

```diff
@@ -66,7 +78,38 @@ internal class PluginServiceIT: BaseIntegrationTest() {
             pluginActionDefinitionKey = "other-test-action",
             actionProperties = Mapper.INSTANCE.get().readTree("""{"someString": "test123"}""")
         )
-        pluginService.invoke(String(), processLink)
+
+        val execution = DelegateExecutionFake.of()
+            .withProcessInstanceId(UUID.randomUUID().toString())
+
+        pluginService.invoke(execution, processLink)
+    }
+
+    @Test
+    @Transactional
+    fun `should invoke an action with a placeholder value on the plugin`() {
```

Comment:
So this is the happy path, what if there is no resolver? This should also be covered by a test.

Reply:
please see https://github.com/valtimo-platform/valtimo-backend-libraries/pull/256/files#r928835211